data_steward/cdr_cleaner/cleaning_rules/null_person_birthdate.py

```diff
@@ -0,0 +1,145 @@
+"""
+Null Person Table Birth Date Fields
+
+In the person table, the fields month_of_birth, day_of_birth, and birth_datetime should be nulled.
+The year_of_birth field should remain unchanged.
+
+Original Issue: DC-1356
+"""
+
+# Python imports
+import logging
+
+# Project imports
+import constants.bq_utils as bq_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import JINJA_ENV
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+
+PERSON = 'person'
+
```

Comment:
Oops, didn't catch this the first time around, you can import the `PERSON` constant with the `JINJA_ENV` so `from common import JINJA_ENV, PERSON`  and get rid of line 22.